lib/yoti/data_type/attribute.rb

```diff
@@ -0,0 +1,26 @@
+module Yoti
+  # Encapsulates profile attribute
+  class Attribute
+    FAMILY_NAME = 'family_name';
+    GIVEN_NAMES = 'given_names';
+    FULL_NAME = 'full_name';
+    DATE_OF_BIRTH = 'date_of_birth';
+    AGE_CONDITION = 'age_condition';
```

Comment:
This is for the getter `age_condition` which will return an attribute object.
It used to be `is_age_verified`, but because this is now returning an object, you cannot just call it that way. Hence the decision to go with `age_condition`

Reply:
wouldn't just `age_verified` work? "Condition" looks a bit weird to me.